src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -391,6 +405,45 @@ private BulkResponse execute() throws Exception {
         }
       }
     }
+
+    private void handleMalformedDoc(BulkResponse bulkRsp) {
+      // if the elasticsearch request failed because of a malformed document,
+      // the behavior is configurable.
+      switch (behaviorOnMalformedDoc) {
+        case WARN:
```

Comment:
very nitpicky but I wouldn't mind matching the order of declaration in the enum (since order doesn't really matter here). `IGNORE, WARN, FAIL`